test/fixtures/dialects/ansi/unaliased_using_subquery.sql

```diff
@@ -0,0 +1,6 @@
+SELECT *
+FROM A_TABLE
+INNER JOIN (
+    SELECT margin
+    FROM B_TABLE
+) USING (SOME_COLUMN)
```

Comment:
Nitpick: Missing newline. Could you add this and regenerate the YAMLs? It may cause someone to be confused in the future, why the file changed (if their editor automatically adds the newline).

Reply:
```suggestion
) USING (SOME_COLUMN)

```

Probably should add this to .editorconfig for .sql files but think there were some sql files we didn't want this for :-(